notification-service/src/test/java/org/java/mentorship/notification/exception/ValidatorHandlerTest.java

```diff
@@ -0,0 +1,38 @@
+package org.java.mentorship.notification.exception;
+
+import jakarta.servlet.http.HttpServletRequest;
+import org.java.mentorship.notification.exception.domain.NotificationServiceError;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.BindingResult;
+import org.springframework.validation.FieldError;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+
+import java.util.Collections;
+import java.util.Objects;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class ValidatorHandlerTest {
```

Comment:
No need for public modifier on the class.
Also, make the validatorHandler variable private